MiniAn/minian_definitions.py

```diff
@@ -53,4 +53,8 @@ class PythonKeys:
     SEED_GROUP_NAME         = "SeedGroupName"
     VIDEO_START             = "VideoStartFrame"
     VIDEO_STOP              = "VideoStopFrame"
+    FNAME_CROSSREG          = "fname_crossReg"
+    H5PATH_IMAGES_CROSSREG  = "h5path_crossRegImages"
```

Comment:
Kapil, could you merge the Maxime's branch and make the changes accordingly

Reply:
Kapil, please don't forget to pull the Maxime's changes, please merge his branch in yours at least once a day.